raster/src/main/scala/geotrellis/raster/histogram/ArrayHistogram.scala

```diff
@@ -79,10 +75,10 @@ class ArrayHistogram(val counts: Array[Int], var total: Int)
     counts(i) += count
   }
 
-  def getItemCount(i: Int) = counts(i)
+  def itemCount(i: Int) = counts(i)
 
   // REFACTOR: use Option
```

Comment:
We should remove these REFACTOR lines, unless what they are trying to do make sense, then we should do them and remove them.